examples/umd-client/index.html

```diff
@@ -18,7 +18,7 @@
   <script src="https://unpkg.com/styled-components@4.1.3/dist/styled-components.min.js" crossorigin></script>
 
   <!-- Load Kepler.gl latest version-->
-  <script src="https://unpkg.com/kepler.gl/umd/keplergl.min.js" crossorigin></script>
+  <script src="https://unpkg.com/kepler.gl@0.2.4/umd/keplergl.min.js" crossorigin></script>
```

Comment:
why add a specific version here?

Reply:
This is to make sure in the future if we make breaking changes to Kepler.gl (require newer version of a library) they will not break downloaded maps

Reply:
Yea, but this is an example right? For easy following example, we should just use latest version.

Reply:
Based on our conversation i will revert this one and look for a better solution as part of a different task